exareme2/controller/services/exareme2/workers.py

```diff
@@ -81,9 +81,9 @@ def __init__(
         self,
         request_id: str,
         context_id: str,
-        worker_tasks_handler: IWorkerAlgorithmTasksHandler,
+        worker_tasks_handler: Exareme2TasksHandler,
```

Comment:
Rename `worker_tasks_handlers` ?

Please check other places as well.

You could also just name it `tasks_handler` if you want.